enhancements/single-node/single-node-openshift-with-workers.md

```diff
@@ -0,0 +1,496 @@
+---
+title: single-node-openshift-with-workers
+authors:
+  - "@omertuc"
+reviewers:
+  - "@romfreiman"
+  - "@eranco74"
+  - "@tsorya"
+  - "@dhellmann"
+  - "@Miciah"
+  - "@bparees"
+  - "@JoelSpeed"
+  - "@staebler"
+  - "@derekwaynecarr"
+approvers:
+  - TBD
```

Comment:
@dhellmann @sdodson who should we add as approvers?

Reply:
There's an API change here, so we need an API approver to look at it. @mfojtik can you help identify someone with time to review this?

I would expect to see someone from the networking team familiar with ingress, too. @knobunc can you help find a reviewer?

Reply:
@tkashem